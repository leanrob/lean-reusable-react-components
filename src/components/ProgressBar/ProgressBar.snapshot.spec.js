import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from './ProgressBar';

test('shows a progress bar', () => {
  const tree = renderer.create(
    <ProgressBar percent={50} width={50}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})