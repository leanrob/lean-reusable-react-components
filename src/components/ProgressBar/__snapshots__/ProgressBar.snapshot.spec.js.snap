// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows a progress bar 1`] = `
<div
  style={
    Object {
      "border": "solid 1px lightgray",
      "width": 50,
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "red",
        "height": 5,
        "width": 25,
      }
    }
  />
</div>
`;
